addon-test-support/scroll.js

```diff
@@ -0,0 +1,47 @@
+import getElementWithAssert from './-private/get-element-with-assert';
+import wait from 'ember-test-helpers/wait';
+
+
+/*
+  Triggers a paint (and therefore flushes any queued up scroll events).
+
+  @method triggerFlushWithPromise
+  @return {RSVP.Promise}
+  @private
+*/
+function triggerFlushWithPromise(){
+  return wait().then(() => {
```

Comment:
So the reason this helper exists is because we actually wrap our usage in of requestAnimationFrame `vertical-collection` with Embe.run: https://github.com/html-next/vertical-collection/blob/master/addon/-private/scheduler/index.js. 

So in order to fully clear out the runloop, we actually need to wait for the next RAF to finish. I'm not sure if this will be helpful to most Ember users, since most of them will just be scheduling things on the standard runloop and there isn't a standard Ember way to do RAF (thus the hack-y scheduler we built).